_posts/2020-07-16-omero-5-6-2.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.2 and OMERO.web 5.7.0
+intro-blurb: This release includes OMERO.server 5.6.2, OMERO.web 5.7.0 and OMERO.py 5.7.1
+---
+
+Today we are releasing OMERO.server 5.6.2 which fixes installaion instructions for CentOS 7, 
```

Comment:
```suggestion
Today we are releasing OMERO.server 5.6.2 which fixes installation instructions for CentOS 7, 
```

Also maybe "the documentation fixes the installation instructions" or were changes also made in the server bundle itself?